### DISCUSSION TITLE
password and recovery help

### DISCUSSION THREAD

**Discussion by JKLcoins** [iotaledger/firefly#2966] [Help & Support]
I am not seeing where I can input my 24 recovery words so that I may access my account. I forgot my password, but still remember my pin. Everywhere I try to do something it requires my password. Where can I input my recovery words?

**Comment by MiKeF05** [comment]
Create a new profile -> recover wallet-> Firefly backup

**Reply by JKLcoins** [reply]
ok thanks.

is there any other way to change a password if you don't remember it?

**Reply by MiKeF05** [reply]
Unfortunatly not. You can change the password in the profile but you need the old password for this process.

**Reply by JKLcoins** [reply]
ok well crap. for whatever reason, i only have 23 words for my recovery. and i can't remember my password. I still have my pin number.  Am I out of options then to access my account and profile?

**Reply by JKLcoins** [reply]
i can still access my profile, but i can't do anything without my password.

**Reply by MiKeF05** [reply]
No. Either you remember the password. Then you can use the old profile. The other way would be to recover your profile with your 24 words. Unfortunatly there are only these 2 options.

**Reply by JKLcoins** [reply]
to be clear, i've lost access to move or do anything with those funds?

**Reply by MiKeF05** [reply]
To send your funds you need the password, yes. Try to remember the password. Maybe you have written it down somewhere. Or get the 24th word. Maybe you have it somewhere written down

**Reply by JKLcoins** [reply]
I did find my stronghold backup file on my computer, but I still need a password for it...
